corehq/apps/ota/utils.py

```diff
@@ -219,3 +224,26 @@ def _inner(request, domain, *args, **kwargs):
 
         return response
     return _inner
+
+
+def can_login_on_device(user_id, device_id):
+    if not device_id or device_id.startswith("WebAppsLogin"):
```

Comment:
Can device ids be set by the submitting device? In other words, would it be possible for someone to make their mobile device(s) ids start with `WebAppsLogin` or be empty and exempt themselves from the limit? Do we care?

Reply:
Yeah good question that I do not know the answer to off the top of my head. I can dig more into this one and get back to you.

Reply:
The official CC Mobile app doesn't allow changing the device ID. Forms submitted by other means can set it to whatever they want. This makes me wonder if we need to account for API submitted forms?